example.py

```diff
@@ -26,37 +26,30 @@ class AwesomeExample(QtGui.QWidget):
     
```

Comment:
A QDialog will center in the screen so maybe it is better to change to 

`class AwesomeExample(QtGui.QDialog)`